calendar/page.js

```diff
@@ -2,18 +2,16 @@
 var grist;
 
 // to keep all calendar related logic;
-let calendarHandler;
-const CALENDAR_NAME = 'standardCalendar';
+var calendarHandler;
+
+var CALENDAR_NAME = 'standardCalendar';
 
 const urlParams = new URLSearchParams(window.location.search);
 const isReadOnly = urlParams.get('readonly') === 'true' ||
   (urlParams.has('access') && urlParams.get('access') !== 'full');
 
-//for tests
-let dataVersion = Date.now();
-function testGetDataVersion(){
-  return dataVersion;
-}
+// for tests
+var dataVersion = Date.now();
```

Comment:
Why are we using var now?

Reply:
It gets added to `window` automatically. But I should also be able to do `window.dataVersion = dataVersion;` if that looks better.